schemas.ts

```diff
@@ -16,7 +16,6 @@ import * as  AirQualityD2C from './schemas/deviceToCloud/air_quality/air_quality
 import * as  ButtonD2C from './schemas/deviceToCloud/button/button.json';
 import * as  DeviceD2C from './schemas/deviceToCloud/device/device.json';
 import * as  FlipD2C from './schemas/deviceToCloud/flip/flip.json';
-import * as  GPSD2C from './schemas/deviceToCloud/gps/gps.json';
```

Comment:
Should there be an entry, mabye `GNSSD2C` for `gnss.json`?

Reply:
AllGPS D2C was doing was was GPS HELLO, so I think there shouldn't be.